projects/demo/src/modules/info/changelog/changelog.style.less

```diff
@@ -48,4 +48,22 @@ markdown {
     & code {
         color: #d45d8c;
     }
+
+    & h3[id^='feat']:before {
+        content: '🚀';
+    }
+
+    & h3[id^='bug']:before {
+        content: '🐞';
+    }
+
+    & h3[id^='deprecations']:before {
+        content: '❌';
```

Comment:
please, use [⚠️](https://emojipedia.org/warning/)

Reply:
It looks in this way.
Is it ok ?

![warning](https://user-images.githubusercontent.com/35179038/183595540-513ed75f-ff85-4e53-b553-6c0a5b1740d8.png)

Reply:
the red color of the icon scares me like it's a breaking change
but in any case, as you wish

Reply:
No problem, I will change

```suggestion
        content: '⚠️';
```